### PR TITLE
feat: support 'ng-true-value' and 'ng-false-value' attrs

### PR DESCRIPTION
**Pull request by DarkNami**

### CODE REVIEW

**Comment by ziscloud** [comment]
@DarkNami welcome your commit, but there are some conflicts, could you please resolve them?

File: dist/angular-bootstrap-toggle.js [hunk 1]
```diff
@@ -94,12 +94,29 @@
...
+                    // Support 'ng-true-value' and 'ng-false-value' attrs
+                    var trueValue = parseConstantExpr($parse, $scope, 'ngTrueValue', $attrs.ngTrueValue, true);
+                    var falseValue = parseConstantExpr($parse, $scope, 'ngFalseValue', $attrs.ngFalseValue, false);
+
+                    function parseConstantExpr($parse, context, name, expression, fallback) {
+                        var parseFn;
+                        if (angular.isDefined(expression)) {
+                            parseFn = $parse(expression);
+                            if (!parseFn.constant) {
+                                throw angular.ngModelMinErr('constexpr', 'Expected constant expression for `{0}`, but saw ' +
```
Comment (bertysentry) [inline dist/angular-bootstrap-toggle.js]: jshint is complaining the line is too long, which breaks the build. Could you please fix this? (yeah, no biggie... ;-) )

File: dist/angular-bootstrap-toggle.js [hunk 2]
```diff
@@ -277,7 +294,7 @@
 						if (self.disabled) { // prevent changing .$viewValue if .disabled == true
 							return false;
 						} else {
-							self.ngModelCtrl.$setViewValue(!self.ngModelCtrl.$viewValue);
+							self.ngModelCtrl.$setViewValue(!angular.equals(self.ngModelCtrl.$viewValue, trueValue) ? trueValue : falseValue);
```
Comment (bertysentry) [inline dist/angular-bootstrap-toggle.js]: Same here: jshint is complaining the line is too long, which breaks the build.

File: dist/angular-bootstrap-toggle.js (hunk 1 above)
Comment (DarkNami) [reply dist/angular-bootstrap-toggle.js]: I fixed. :)

File: dist/angular-bootstrap-toggle.js (hunk 2 above)
Comment (DarkNami) [reply dist/angular-bootstrap-toggle.js]: Fixed.

**Pull request merged**